webrtc/videoroom/assets/src/room_ui.ts

```diff
@@ -132,6 +129,20 @@ export function setErrorMessage(
   }
 }
 
+export function replaceStream(oldStreamId: String, newStream: MediaStream) {
+  const grid = document.getElementById("videos-grid");
+  var videoElements = grid?.getElementsByTagName("video")!;
+  for (var i = 0; i < videoElements.length; i++) {
```

Comment:
Use `let` if variable's reference is meant to be replaced or `const` otherwise.

Reply:
`let` in this case

Reply:
And more appropriate would be:
```js
const oldVideo = videoElements.find(video => video.id === oldStreamId);
if (oldVideo) ...
```

Reply:
I can't use `find` on `HTMLCollectionOf<HTMLVideoElement>`